src/commands/force/source/clean.ts

```diff
@@ -160,28 +132,28 @@ function markContents(targetDir: string, ignore: Ignore) {
         if (stat.isDirectory()) {
           markContents(filePath, ignore);
         } else {
-
           if (!ignore.ignores(filePath)) {
-            if (!filePath.endsWith('-meta.xml')) {
```

Comment:
@ben-faber-ccc  This is basically the change...  We used to delete files except for the `-meta.xml`.  

I think the delete marker was added later. I'm not sure why we didn't just make it mark everything at that point.